packages/view/src/components/Detail/Detail.util.ts

```diff
@@ -0,0 +1,34 @@
+import type { GlobalProps } from "types/global";
+
+import type { CommitNode } from "types/NodeTypes.temp";
+
+type GetTargetCommit = GlobalProps & { id: string };
+
+export const getTargetCommit = ({ data, id }: GetTargetCommit) => {
+  if (data?.length === 0) return undefined;
+  const flatCommitNode: CommitNode[] = data
+    .map((clusterNode) => clusterNode.commitNodeList)
+    .flat();
+  const [target] = flatCommitNode.filter(
+    (commitNode) => commitNode.commit.id === id
```

Comment:
```suggestion
    ({ commit } ) => commit.id === id
```
요런식으로 사용하는 것에 대해서는 어떻게 생각하시나요? @_@

Reply:
오 좋은 것 같습니다.
둘의 장단점은 있을 것 같아요. 

윗쪽은 변수명을 넣으면서 어떤 용도로 사용 되는지에 대한 표기를 할 수 있을 꺼고
아랫쪽은 코드가 짧아지고 명확해지겠네요 : -)

Reply:
맞습니다 ㅎㅎ 저도 디스트럭쳐링할당 정말 좋아하는데요, 간과했던것 같습니다 ㅎㅎ